packages/react/src/actions/bitcoin/bitcoinApiConfig.ts

```diff
@@ -1,31 +1,99 @@
-export interface BitcoinApiConfig {
-  mainnet: string;
-  testnet: string;
-  name: string;
-}
+import { ApiConfig, ApiResponse } from '../../types/bitcoin.js';
 
-export const bitcoinApiConfig: Record<string, BitcoinApiConfig> = {
-  blockstream: {
-    mainnet: 'https://blockstream.info',
-    testnet: 'https://blockstream.info/testnet',
-    name: 'blockstream',
+export const APIs: ApiConfig[] = [
+  {
+    name: 'btcscan',
+    url: {
+      balance: (address: string) => `https://btcscan.org/api/address/${address}`,
+      transaction: (txHash: string) => `https://btcscan.org/api/tx/${txHash}/status`,
+    },
   },
-  mempool: {
-    mainnet: 'https://mempool.space',
-    testnet: 'https://mempool.space/testnet',
-    name: 'mempool',
+  {
+    name: 'blockchain.info',
+    url: {
+      balance: (address: string) => `https://api.blockchain.info/haskoin-store/btc/address/${address}/balance`,
+      transaction: (txHash: string) => `https://api.blockchain.info/haskoin-store/btc/transaction/${txHash}`,
+    },
   },
-};
+  {
+    name: 'blockcypher',
+    url: {
+      balance: (address: string) => `https://api.blockcypher.com/v1/btc/main/addrs/${address}/balance`,
+      transaction: (txHash: string) => `https://api.blockcypher.com/v1/btc/main/txs/${txHash}`,
+    },
+  },
+];
```

Comment:
can also make them a record so that you don't have to .find the api everytime. 
small refactor - optional